README.md

```diff
@@ -1,25 +1,38 @@
 Chitter Challenge
```

Comment:
Nice well-written README, only the part with technologies you used is missing from it.